rke/k8s_rke_system_images.go

```diff
@@ -2843,6 +2878,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.4"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled in Rancher v2.3.8
```

Comment:
Enabled out of band for Rancher v2.3.8 with Rancher v2.4.5

Reply:
This still isnt updated